doc/rst/_static/style.css

```
@@ -8,6 +8,16 @@
     font-weight: normal;
 }
 
+body {
+	font-family: "Atkinson Hyperlegible", sans-serif;
+    font-size: 1.05em;
```

One small whitespace nitpick
```suggestion
    font-family: "Atkinson Hyperlegible", sans-serif;
    font-size: 1.05em;
```